blueocean-testing/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IF we really want a discrete testing utils module - probably best done separately as a PR